hack/generate-iam-from-credrequest.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+set -euo pipefail
+
+CR_FILE="${1:-operator-credentials-request.yaml}"
+POLICY_FILE="${2:-operator-permission-policy.json}"
+YQ_BIN="go run github.com/mikefarah/yq/v4"
```

Comment:
Does the user already have access to the `go` binary?

Reply:
This script is run as a part of `make generate` rule which is used during the building of the operator binary. So, it's assumed that the user has `go` toolchain.